docs/multi_file_content_management.md

```diff
@@ -0,0 +1,25 @@
+# Multi-file content management
+
+Multi-file content management is a set of functionalities that enables you to upload, move and delete multiple files in bulk.
```

Comment:
I wonder if it shouldn't be `enable` without `s` - depends if it refers to "Multi-file content management" or "set of functionalities". Interesting. 🤔

Reply:
It's all good :)

Reply:
OK 😄 Sorry for unnecessary comment! :)

Reply:
It wasn't unnecessary, both ways are correct, so you weren't wrong 😄